1_OCaml/TP/TP5.ipynb

```diff
@@ -19,6 +19,132 @@
     "4. Tester si un tableau est trié par ordre croissant."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.             if i &lt; t.((Array.length t) - 1) then true
</pre><p>A revoir, ça ne fait pas ce qui est demandé.</p><p>D'ailleurs l'erreur de typage est plutôt claire : un type unit (c'est-à-dire rien) est attendu dans le corps de la boucle for, et ici, c'est du bool.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>